source/index.md

```diff
@@ -600,9 +602,46 @@ object][ApplePay-PaymentToken] for more information.
   subsequent recurring authorization.
 </p>
 
-##### Method: `mobilepayonline`
 
-<dl class="dl-horizontal">
+---
+##### **Method**: `googlepay`
+
+To accept a payment using Google Pay, apart from the complete payment method
+token and merchant ID, the derived shared secret is required. This must be
+derived according to the documentation.
+
+The cryptography involved is described [here][GooglePay-PaymentCryptography].
+
+<dl>
+  <dt>googlepay[token]</dt>
+  <dd>[:json:] <br /> Raw payment method token as received in response from Google. UTF-8
+    encoded serialization of a JSON dictionary.</dd>
+  <dt>googlepay[shared_secret]</dt>
+  <dd>
+    The shared secret derived from the ephemeral public key and your private key, Base64 encoded.
```

Comment:
The description is correct, but just slightly different from the one for `applepay`. I do not have a strong preference as to which to chosen, I just think they should "agree more" as they are very similar under the hood 🙂

Reply:
Well. They are similar, but the secret passed here is not a  symmetric AES.
Combined with the (Google signed) ephemeral public key it is used to create the symmetric AES key and a MAC key.

I don't know. Thinking of going with something like:
```html5
  <dt>googlepay[shared_secret]</dt>
  <dd>[:base64:]<br />The shared secret derived from the ephemeral public key and
    your private key.</dd>
```

Reply:
Ahh, yes, right. I like your thinking! 👍